WalletWasabi.Packager/Tools.cs

```diff
@@ -44,18 +44,18 @@ public static void ClearSha512Tags(string pathToSearch)
 			}
 		}
 
-		//public static void RemoveSosDocsUnix(string pathToSearch)
-		//{
-		//	// ToDo ?: Must remove from the ui.deps.json, too otherwise binary will fail.
-		//	// Leave it to not do anything for now.
-		//	string[] files = Directory.GetFiles(pathToSearch, "sosdocsunix.txt", SearchOption.AllDirectories);
-		//	if (files is null) return;
+		////public static void RemoveSosDocsUnix(string pathToSearch)
```

Comment:
Delete the whole function.

Reply:
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Packager/Program.cs#L511
I didn't delete it because there is that commented code and I thought it was left on purpose, should I delete them anyway?

Reply:
I would ask the owner of that line. Can you do blame on that line and see who commented that out?

Reply:
Feel free to delete this uncommented code. (And of course the unused `RemoveSosDocsUnix` function created because of this.)

Reply:
Done.